mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -513,7 +536,8 @@ private void assertIsOwner(MailboxSession mailboxSession, MailboxPath mailboxPat
         }
     }
 
-    private List<MailboxRenamedResult> doRenameMailbox(Mailbox mailbox, MailboxPath newMailboxPath, MailboxSession session, MailboxMapper mapper) throws MailboxException {
+    private List<MailboxRenamedResult> doRenameMailbox(Mailbox mailbox, MailboxPath newMailboxPath, MailboxSession session,
```

Comment:
This change here is not needed, can you revert it?

Reply:
ok